site/guide/4-requests.md

```diff
@@ -52,7 +52,7 @@ your application explicitly handles.
 
 ### Reinterpreting
 
-Because browsers can only send `GET` and `POST` requests, Rocket _reinterprets_
+Because HTML forms are only allowed to use `GET` and `POST` HTTP methods, Rocket _reinterprets_
```

Comment:
I think the following makes the rationale more obvious than either of these:

"Because HTML forms can only be sent as `GET` or `POST` requests unless client-side scripts are used,"

Reply:
I agree, that works as well, no preference on my side. The confusion from the original wording would be gone either way.